bundles/org.openhab.binding.systeminfo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -254,34 +235,34 @@
 	</channel-type>
 
 	<channel-type id="remainingTime">
-		<item-type>Number</item-type>
+		<item-type>Number:Time</item-type>
 		<label>Remaining Time</label>
-		<description>Remaining time in minutes</description>
-		<state readOnly="true" pattern="%.1f Minutes"/>
+		<description>Remaining battery time</description>
+		<state readOnly="true" pattern="%f %unit%"/>
 		<config-description-ref uri="systeminfo:channels:mediumpriority"/>
 	</channel-type>
 
 	<channel-type id="remainingCapacity">
-		<item-type>Number</item-type>
+		<item-type>Number:Dimensionless</item-type>
 		<label>Remaining Capacity</label>
-		<description>Remaining capacity in percent</description>
-		<state readOnly="true" pattern="%.1f %%"/>
+		<description>Remaining battery capacity</description>
+		<state readOnly="true" pattern="%f %%"/>
 		<config-description-ref uri="systeminfo:channels:mediumpriority"/>
 	</channel-type>
 
-	<channel-type id="load_process">
-		<item-type>Number</item-type>
-		<label>Load</label>
-		<description>Load in percent</description>
-		<state readOnly="true" pattern="%.1f %%"/>
-		<config-description-ref uri="systeminfo:channels:highpriority_process"/>
+	<channel-type id="loadHigh">
+		<item-type>Number:Dimensionless</item-type>
+		<label>System Load</label>
+		<description>System Load in a given time</description>
+		<state readOnly="true" pattern="%f %%"/>
+		<config-description-ref uri="systeminfo:channels:highpriority"/>
 	</channel-type>
 
-	<channel-type id="loadAverage" advanced="true">
-		<item-type>Number</item-type>
-		<label>Load Average</label>
-		<description>Load as a number of processes for the last 1,5 or 15 minutes</description>
-		<state readOnly="true" pattern="%.1f"/>
+	<channel-type id="loadMedium">
+		<item-type>Number:Dimensionless</item-type>
+		<label>System Load</label>
+		<description>System Load in a given time</description>
+		<state readOnly="true" pattern="%f %%"/>
```

Comment:
I just stumbled over this change and don't think this is correct, at least according to the load numbers given in UNIX computing. AFAIK these values are no percentages but rather the average number of processes that were to handle in the given period and can therefore be higher than 1, see also [wikipedia](https://en.wikipedia.org/wiki/Load_(computing)#Interpretation) on how to interpret these.
However, I have to admit that I don't know how it is handled in this binding.

Reply:
I took a look into openhab documentation and found no hint, that percentage is limited to 100

Reply:
That's correct, percentage is not limited to 100%. What @boehan means is that the value is not in %, but some average of process count. So, I would simply remove the %-sign from the state pattern and leave it without a unit.

Reply:
Disagree. Linux commands htop and top shows CPU untilization as %. And yes, the calculation way is different between linux and windows. Additionally, OSHI returns the values, as far i've understood, in %.

Reply:
Ok, but the code we discuss here is for the system load, not the CPU utilization.